PlayerAdministration.cs

```diff
@@ -159,12 +164,14 @@ public string Add(CuiInputField aInputField, string aParent = Cui.ParentHud, str
                 if (string.IsNullOrEmpty(aName))
                     aName = CuiHelper.GetGuid();
 
-                if (aInputField == null) {
+                if (aInputField == null)
+                {
```

Comment:
Please try to avoid changing the coding style, this "modern day Microsoft look" looks really crappy to me, I'm more into the style I used throughout. :)

Reply:
I actually 100% agree with you, it was a newly setup visual studio. I tend to use intellij more in my day to day coding, so not sure where to change the linting. Will have a search and adjust.